{
  "test_mynames.py::test_get_string_raises_exceptions": true
}